message-index/messages/GHC-53786/case-expr-in-case-pattern/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Case expression in case pattern
+---
+
+The pattern in the case expression `case x of ...` contains a case expression.
```

Comment:
```suggestion
title: Case expression in case pattern
---

The pattern *being matched upon* in the case expression `case x of ...` itself contains a case expression.
```